src/components/messageCanvas/messageCanvas.stories.tsx

```diff
@@ -81,7 +64,7 @@ const profileString = `(message: ThreadableMessage) => {
   }`
 
 const getProfileComponent = (message: ThreadableMessage) => {
-  if (message.sender.includes('agent')) {
+  if (message.sender.name?.includes('agent')) {
     return <Icon name="smart_toy" />
   } else {
     return <Icon name="account_circle" />
```

Comment:
Since we remove sender name from the component, should we update this to `getProfileIcon` and add another function to add profile name? Same should be done for MessageSpace.
```
function getProfileComponent(message: ThreadableMessage) {
  return (
    <>
      {getProfileIcon(message)}
      <Typography variant="body1" color="text.secondary">
        {message.sender.name}
      </Typography>
    </>
  )
}
```

Reply:
I was thinking that would be upto the application developer whether they want just icon or icon with name or just name. We should not control that too much as it does not give much flexibility

Reply:
I'm just thinking it would be nice to match with design. We could also do that in a follow-up too.

Reply:
ack. will do in a follow-up since that is just a change to the examples in docs